ui/test/nuxeo-document-suggestion.test.js

```diff
@@ -40,11 +40,20 @@ function getSuggestions(suggestionWidget, timeout = 1000) {
 
 // Mock router
 const router = {
-  browse: () => '#',
-  user: () => '#',
-  group: () => '#',
+  browse: (path) => path,
+  document: (uid) => `/doc/${uid}`,
 };
 
+function setNuxeoRouterKey(entityType, value) {
```

Comment:
remark: we really need  a `window.Nuxeo.UI.config.set('...')` or something...

Reply:
+1